### PR TITLE
TIP-276: launch the Mongo specs on Travis and remove HHVM

### PR DESCRIPTION
**Pull request by jjanvier**
What's done here:
 * fixes the specs with Mongo and PHP7
 * launches all the specs (even Mongo) on Travis-CI
 * removes HHVM from the Travis matrix 

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added Behats                      | -
| Changelog updated                 | -
| Review and 2 GTM                  |
| Micro Demo to the PO (Story only) |
| Migration script                  | -
| Tech Doc                          | -

### CODE REVIEW

File: phpspec-fix [hunk 1]
```diff
@@ -1,16 +0,0 @@
-#!/bin/sh
-
-# PhpSpec run fix that prevent suite containing skipped example to make travis build fail
-
-touch app/config/parameters_test.yml
```
Comment (nidup) [inline phpspec-fix]: :fire: :+1:

**Comment by skeleton** [comment]
gtm

File: .travis.yml [hunk 2]
```diff
@@ -33,7 +33,7 @@ install:
 
 script:
     - ./bin/phpunit -c app/phpunit.travis.xml --testsuite PIM_Unit_Test
-    - ./phpspec-fix
+    - ./bin/phpspec run
```
Comment (aRn0D) [inline .travis.yml]: Don't we need this fix anymore ?

File: phpspec.yml.dist [hunk 3]
```diff
@@ -184,6 +184,3 @@ suites:
         psr4_prefix: Oro\Bundle\FilterBundle
         spec_path: src/Oro/Bundle/FilterBundle
         src_path: src/Oro/Bundle/FilterBundle
-
-extensions:
-    Akeneo\SkipExampleExtension: ~
```
Comment (pierallard) [inline phpspec.yml.dist]: Maybe you have to drop the class too ?

**Comment by pierallard** [comment]
Same question than @aRn0D then GTM

File: .travis.yml (hunk 2 above)
Comment (jjanvier) [inline .travis.yml]: seen IRL, I dropped a custom script that changed that return code of phpspec

File: phpspec.yml.dist (hunk 3 above)
Comment (jjanvier) [inline phpspec.yml.dist]: no this class is provided by the vendor phpspec-skip-extension (made by Akeneo ;) )

**Comment by nidup** [comment]
GTM

**Comment by aRn0D** [comment]
GTM

**Pull request merged**